drivers/nec/display/display.cr

```diff
@@ -0,0 +1,397 @@
+module Nec; end
+module Nec::Display; end
+
+require "placeos-driver/interface/powerable"
+require "placeos-driver/interface/muteable"
+require "placeos-driver/interface/switchable"
+
+class Nec::Display::All < PlaceOS::Driver
+  include Interface::Powerable
+  include Interface::AudioMuteable
+
+  enum Inputs
+    Vga          = 1
+    Rgbhv        = 2
+    Dvi          = 3
+    Hdmi_set     = 4
+    Video1       = 5
+    Video2       = 6
+    Svideo       = 7
+    Tuner        = 9
+    Tv           = 10
+    Dvd1         = 12
+    Option       = 13
+    Dvd2         = 14
+    Display_port = 15
+    Hdmi         = 17
+    Hdmi2        = 18
+    Hdmi3        = 130
+    Usb          = 135
+
+    def to_s : String
```

Comment:
Hey @KimBurgess , I'm not sure what you mean by this

Reply:
This was in reference to `Inputs#to_s`. That serialisation is specific to how an `Input` should appear as part of the outgoing command. To keep this all grouped together, this can be performed within your `do_send` method.